### DISCUSSION TITLE
部署说明

### DISCUSSION THREAD

**Discussion by nunamia** [BigBrotherTrade/trader#4] [Q&A]
部署：
1.环境配置
2.第三方api key申请
3.调试

**Comment by xjf01793** [comment]
@nunamia  请教一下，我的为什么报错：panel.models.Strategy.DoesNotExist: Strategy matching query does not exist.

**Comment by xjf01793** [comment]
@nunamia 能把的部署步骤提供一下吗？谢谢！1

**Comment by ShangyiAlone** [comment]
![image](https://github.com/user-attachments/assets/7cb3d131-7d8f-4626-b460-7ffdb31a9d8c)
这边爆错这个包无法下载，请问应该怎么处理呢？
